gulpfile.js

```diff
@@ -149,7 +149,7 @@ gulp.task("build:download-scanners", () => {
     );
 
     streams.push(
-      downloadOrCopy(scanner.dotnetUrl)
+      downloadOrCopy(process.env.SCANNER_NET_DOTNET_LOCATION ?? scanner.dotnetUrl)
```

Comment:
Thankfully, we already have the `downloadOrCopy` utility that can either download remote files or copy local files

Reply:
NP: this refers to the scanner filename, 
e.g.
`sonar-scanner-${msBuildVersion}-net-framework.zip`
`sonar-scanner-${msBuildVersion}-net.zip`

and could be aligned
`SCANNER_NET_FRAMEWORK_LOCATION`
`SCANNER_NET_LOCATION`